app/javascript/pages/IndividualEvents/query.gql

```diff
@@ -2,7 +2,7 @@
 #import "fragments/OfficeEntry.gql"
 #import "fragments/OrganizationEntry.gql"
 
-query IndividualEventsQuery {
+query MyEventsQuery {
```

Comment:
Since this query is not only about individual events, should we move/rename this file?